composer.json

```diff
@@ -69,10 +69,10 @@
   "require-dev": {
     "composer/composer": "^1.10.0",
     "phpunit/phpunit": "^8",
-    "fzaninotto/faker": "~1.9.0",
+    "fzaninotto/faker": "dev-master",
```

Comment:
Yes, this solution is kind of not cool. But as far I can see, PHP 8 support was just monkeypatched into faker just on the master branch. As the development is stopped, I guess it is fine to keep this until removing faker from here.